name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/panes/AccountTransactionsPane.java

```diff
@@ -364,6 +366,31 @@ public Color getForeground(Object element)
         ColumnViewerSorter.createIgnoreCase(e -> ((AccountTransaction) e).getSource()).attachTo(column); // $NON-NLS-1$
         transactionsColumns.addColumn(column);
 
+        column = new Column("9", Messages.ColumnTaxes, SWT.RIGHT, 80); //$NON-NLS-1$
+        column.setLabelProvider(new ColumnLabelProvider()
+        {
+            @Override
+            public String getText(Object e)
+            {
+                AccountTransaction t = (AccountTransaction) e;
+                return Values.Money.format(Money.of(t.getCurrencyCode(), t.getTaxes()), client.getBaseCurrency());
+            }
+
+            @Override
+            public Color getForeground(Object element)
+            {
+                return colorFor((AccountTransaction) element);
+            }
+        });
+        ColumnViewerSorter
+                        .create(element -> ((TransactionPair<?>) element).getTransaction() instanceof AccountTransaction
```

Comment:
This will result in a ClassCastException because the items are actually of type AccountTransaction